www/kasj08/semestralka2/scripts/main.js

```diff
@@ -0,0 +1,504 @@
+var API_ID = 'd1c9a91ea65443af90946fde02fdda64';
+var API_SECRET = '26bbf4fad9384fd4bb3543649ade8b05';
+var REDIRECT_URI = 'http://localhost:5500';
```

Comment:
pridej prosim do redirect_uri whitelistu `http://127.0.0.1:5500/www/kasj08/semestralka2/` a `https://fcp.vse.cz/4IZ268/2018-2019-ZS/www/kasj08/semestralka2/`, jinak to nebude fungovat na local a na fcp serveru po merge

Reply:
přidáno